Tests/Controller/ControllerTest.php

```diff
@@ -286,7 +286,7 @@ public function testGetTranslationsWithUnderscoredLocale()
     public function testGetTranslationsOutsideResourcesFolder()
     {
         if (Kernel::VERSION_ID < 20800) {
-            return;
+            $this->markTestSkipped('Version do not support framework.paths configuration');
```

Comment:
I would say ``This Symfony version``, to explain which version you are talking about